### ISSUE TITLE
Find scripts inside directories

### ISSUE THREAD

**Issue by jantari** [ironmansoftware/powershell-universal#258]
**Is your feature request related to a problem? Please describe.**

We have a pre-existing git Repository with scripts meant for automation, it contains ~85 scripts and they're organized into a directory structure based on their usecase/product they relate to. Currently when I sync this repository to Universal Automation, it finds exactly 0 scripts because it doesn't appear to look through the directories.

**Describe the solution you'd like**
When syncing our git repository, Universal Automation should find all scripts inside all subdirectories.

**Describe alternatives you've considered**
Make a disorganized mess of a repository with 85+ scripts directly in the root-directory. Not an option, I'd be fired.

**Additional context**
Tested with Universal Automation 2.0.3 on Windows Server 2019

**Comment by rwidmark** [comment]
+1 on this, it should bee looking recursive two levels at least.
Here it's important that it don't drag down performance but I guess it looks after new scripts when the service are started?

**Comment by adamdriscoll** [comment]
Right now, PSU doesn't discover scripts at all. I've updated the 2.1.0 admin console to support nested folders. You'll be able to update the scripts.ps1 to include relative paths. 

This is how it looks in the admin console. 

![image](https://user-images.githubusercontent.com/1256531/123708932-d72d4f00-d831-11eb-96c6-93e433a3af24.png)

This is what my scripts.ps1 looks like. 

```
New-PSUScript -Name "Test.ps1" -Path "Test.ps1" -InformationAction "Continue"
New-PSUScript -Name "DevOps2" -Path "ops\devops\myscript.ps1" -InformationAction "Continue"
```

**Comment by jantari** [comment]
Ok so if we wanted "discovery-behavor" we could do sth like:

    # scripts.ps1
    Get-ChildItem -Recurse -File -Filter '*.ps1' |
        Foreach-Object {
            New-PSUScript -Name $_.BaseName Path $_.FullName -InformationAction "Continue"
        }

?

**Comment by adamdriscoll** [comment]
Yep. That would work. Just released 2.1.0 and you can give it a shot.

**Comment by jantari** [comment]
Thank you, I just tried it out and it *mostly* worked as expected.

The only thing I had to change was to add an explicit path for `Get-ChildItem` because by default it runs inside the PowerShell Universal program directory where it doesn't find any of our scripts ofc. I sort of  expected it to run from its `.universal` directory where it's located, but anyway I ended up using it like this:

```powershell
# scripts.ps1
Get-ChildItem -LiteralPath "D:\UniversalAutomation\Repository" -Recurse -File -Filter '*.ps1' |
    Foreach-Object {
        New-PSUScript -Name $_.BaseName -Path $_.FullName -InformationAction "Continue"
    }
```
